app/assets/javascripts/Results/file_selector.js

```diff
@@ -3,3 +3,10 @@ function with_annotations() {
   document.getElementById('download_include_annotations').value =
         document.getElementById('include_annotations').checked ? 'true' : 'false';
 }
+
+function annotations_included() {
+
+  if (document.getElementById('include_annotations').checked) {
+    alert("Warning! Annotations on PDFs are not currently downloaded.");
```

Comment:
@jeremygoh This string should be localized. This means you'll want to put this function inside an ERB file like `app/views/results/marker/_boot.js.erb` and edit the `config/locales/*.yml` files